### DISCUSSION TITLE
GitHub India

### DISCUSSION THREAD

**Discussion by Monukumar193** [github/india#842] [GitHub Constellation 🇮🇳 2022 Q&A]
Hello Folks, thanks for giving me important info. Amazing session is going.
Thanks Indian Developers, GitHub Stars 🙏🙏🙏

**Comment by sarkardeep** [comment]
I attended the Constellation 2022 two days now and 
The event was a blast and there are a number of things that caught my eye.
1. Seamless flow of events
2. Great sessions and fantastic topics
3. Unique hosting style
4. live tweets and so many new learning 
5. Animations and Visual effects for better understanding.
I'm super excited for the remaining part of this.

**Comment by ishanavasthi** [comment]
Am happy to see an Indian version of these events with Indian CEOs
This event is amazing.
